bool connected
bool motors_enabled
bool safety_range_enabled
int64 locomotion_mode
int64 vel
int64 steering
